#!/usr/bin/env python
PACKAGE = "teamplay"

import roslib;
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("setpieceExecuteTimeout", int_t, 0, "time-out in milliseconds", 0, 0, 200000)
gen.add("penaltyExecuteTimeout", int_t, 0, "time-out in milliseconds", 0, 0, 200000)
gen.add("minKickDistanceKicked", double_t, 0, "minimum setpiece distance kicked in meters for receivers", 0, 0, 18)
gen.add("minPenaltyDistanceKicked", double_t, 0, "minimum penalty distance kicked in meter",0 , 0, 18)
gen.add("minOwnKickoffDistanceKicked", double_t, 0, "minimum setpiece distance kicked in meters for kicker", 0, 0, 18)
gen.add("stimulatePassingEnabled", bool_t, 0, "stimulate passing and dribbling (rule since 2017) is enabled", True)

exit(gen.generate(PACKAGE, "teamplayNode", "teamplayRules"))
